client/src/main/java/org/evosuite/coverage/rho/RhoTestSuiteSecondaryObjective.java

```diff
@@ -40,10 +41,13 @@ private double getRhoFitnessValue(TestSuiteChromosome suite) {
      * {@inheritDoc}
      */
     @Override
-    public int compareChromosomes(TestSuiteChromosome suite1, TestSuiteChromosome suite2) {
-        double c1 = this.getRhoFitnessValue(suite1);
-        double c2 = this.getRhoFitnessValue(suite2);
+    public double compareChromosomes(TestSuiteChromosome suite1, TestSuiteChromosome suite2) {
+        double c1 = FitnessFunction.normalize(this.getRhoFitnessValue(suite1));
+        double c2 = FitnessFunction.normalize(this.getRhoFitnessValue(suite2));
 
+        return c1 - c2;
+
+        /*
```

Comment:
Remove the following.

Reply:
I'm afraid I can't remove this. We made this change because we decided that it would be better to normalize the secondary criteria. In our case, we wanted the smelliness of each test smell to vary between 0 and 1.
https://github.com/jose/smell-free-tests-evosuite/commit/677f74d42bb1fe2a2a7c4daa58b1f341cdf7f213

Regarding "RhoTestSuiteSecondaryObjective.java", instead of just returning 0 or 1, we decided to calculate the smelliness of each test suite, normalize the respective values, and return the difference between the two. We calculate the difference because we considered that a test suite that is a lot less smelly than the other should have more weight than a test that is just a bit less smelly than another.

Removing this would most likely break our test smell optimization process. At the very least, there would be some unexpected results...

Is this change likely to break the rest of the code? Or are there any chances that we can keep it like this?